config/application.rb

```diff
@@ -50,5 +50,6 @@ class Application < Rails::Application
       production: "www.getctc.org"
     }
     config.middleware.use Middleware::CleanupMimeTypeHeaders
+    config.default_tax_year = 2020
```

Comment:
consider renaming to `ctc_default_tax_year` since there isn't really a default year for gyr services